docs/pages/eas-update/rollouts.mdx

```diff
@@ -1,32 +1,76 @@
 ---
 title: Rollouts
-description: Incrementally deploy updates on a new branch to a channel.
+description: Incrementally deploy updates to your users.
```

Comment:
```suggestion
description: Learn how to incrementally deploy updates to your users by using a rollout mechanism.
```

Let's expand this description to add the context about using a rollout mechanism.